lib/src/folding/parsers/spaces_foldable_block_parser.dart

```diff
@@ -0,0 +1,109 @@
+import 'dart:collection';
+
+import 'package:charcode/ascii.dart';
+import '../foldable_block.dart';
+import '../foldable_block_type.dart';
+
+import 'abstract.dart';
+
+///A parser for foldable blocks from spaces count at start of each line.
```

Comment:
In English, it's "space count".
Also "indentation" is shorter than "space count at start of".

Add a space after `///` please.